### DISCUSSION TITLE
Abandoned Messages.

### DISCUSSION THREAD

**Discussion by gregeeh** [wallabag/wallabag#6722] [Q&A] [answered]
Hi all,

Installed Wallabag 2.6.1 in a docker container on my OMV NAS.  Noticed lots of package abandoned messages in the logs.  Anything to worry about?

TIA

**Comment by j0k3r** [comment, marked as answer]
We are in the process to handle them in the next version 2.7.0, etc.

**Reply by gregeeh** [reply]
Thank you, once again.
